src/xcode/ENA/ENA/Source/Scenes/ENSetting/EUSettings/EUSettingsViewController.swift

```diff
@@ -40,14 +20,36 @@ class EUSettingsViewController: DynamicTableViewController {
 		fatalError("init(coder:) has not been implemented")
 	}
 
-	// MARK: - View life cycle methods.
+	// MARK: - Deinit
+
+	deinit {
+		if let observer = applicationDidBecomeActiveObserver {
+			NotificationCenter.default.removeObserver(observer)
+		}
+		appConfigCancellable?.cancel()
+	}
+
+	// MARK: - Overrides
 
 	override func viewDidLoad() {
 		super.viewDidLoad()
 		setupView()
 	}
 
-	// MARK: - View setup methods.
+	// MARK: - Internal
+
+	var appConfigurationProvider: AppConfigurationProviding
+
+	enum CustomCellReuseIdentifiers: String, TableViewCellReuseIdentifiers {
+		case countries = "LabeledCountriesCell"
```

Comment:
Just wondering don't we store cell identifiers as constants?

Reply:
yes sometime this gets used too. I reuse the cell from an other view, will try if it's needed